infrastructure/storage-provisioning/README.md

```diff
@@ -85,3 +85,25 @@ After that you can delete test with commands
 $ kubectl delete -f examples/wordpress.yaml
 $ kubectl delete -f examples/mysql.yaml
 ```
+
+## Rook NFS
+To use the user's personal storage we decided to use the NFS server provided by Rook-Ceph ([here](https://rook.io/docs/rook/v1.10/Storage-Configuration/NFS/nfs/)).
+
+With the `CephNFS` CRD we configure a NFS-Ganesha server to provide the access to the user's personal storage via NFS inside the cluster.
+
+A `StorageClass` is used for the dynamic provisioning of the user's storage. In this `StorageClass` is specified, via specific parameters for the Rook Operator, to create also a NFS share to which the user can later connect to via the NFS protocol.
+
+To create the necessary resources:
+```bash
+$ kubectl create -f nfs-manifests/nfs.yaml
+$ kubectl create -f nfs-manifests/storageclass.yaml
+```
+
+Be aware that to use this feature the system must be configured with:
+- Rook v1.9.1 or greather
+- Ceph v16.2.7 or greather
+
+### Using the NFS service
+Using the NFS service inside VirtualMachines requires the `nfs-common` package for Debian based linux OSs and the `nfs-utils` for CentOS based VMs.
+
+On Container instances the DNS resolution for the service name is done on the host machine that doesn't have access to the Kubernetes cluster's DNS server. So to make this solution work the host machine must be able to resolve the DNS name for the NFS server adding the record to the host's `/etc/resolv.conf` either manually or with an automated process.
```

Comment:
```suggestion
The DNS resolution for the service name on Container instances is done on the host machine, which does not have access to the Kubernetes cluster's DNS server.
To allow the host machine to resolve the DNS name for the NFS server you need to add the proper record to the host's `/etc/resolv.conf`, either manually or with an automated process (e.g., cloud-init).
```
NOTE: which record do I have to add? I'm kind of lost here.

Reply:
I will add a more detailed explanation